src/pybind/mgr/restful/module.py

```diff
@@ -495,7 +495,11 @@ def get_mons(self):
         # Add more information
         for mon in mon_map_mons:
             mon['in_quorum'] = mon['rank'] in mon_status['quorum']
-            mon['server'] = self.get_metadata("mon", mon['name'])['hostname']
+            try:
+                mon['server'] = self.get_metadata("mon", mon['name'])['hostname']
+            except:
+                self.log.error("unknown server, it maybe down")
```

Comment:
why do you think it's down? i think if it is down, it won't show up in the monmap.